docs/LTS.md

```diff
@@ -0,0 +1,24 @@
+<a name="lts"></a>
+# Long Term Support
+
+Fastify's Long Term Support (LTS) is provided according the schedule laid
+out in this document:
+
+1. Major releases, "X" release of [semantic versioning][semver] X.Y.Z release
+versions, are supported for a minimum period of six months from their release
+date. The release date of any specific version can be found at
+[https://github.com/fastify/fastify/releases](https://github.com/fastify/fastify/releases).
+
+1. Major releases will receive security updates for an additional six months
```

Comment:
`2.`

Reply:
It gets auto corrected by the Markdown renderer.

Reply:
FYI, the reason for letting the Markdown renderer handle it is to make it easier to rearrange points in the future.